IBAnimatable/ActivityIndicatorAnimationBallRotateChase.swift

```diff
@@ -30,9 +30,8 @@ public class ActivityIndicatorAnimationBallRotateChase: ActivityIndicatorAnimati
 
 
 // MARK: - Setup
-
+// swiftlint:disable variable_name_min_length
```

Comment:
Won't it be better to use `swifltint:disable:next variable_name_min_length` instead of `{disable,enable}`?